wallet/src/transaction/dump.rs

```diff
@@ -84,6 +84,8 @@ impl Dump {
             let builder = builder.set_witnesses(&witnesses);
             let tx = builder.set_payload_auth(&());
             self.outputs.push(tx);
+        } else {
+            self.ignored.extend_from_slice(self.inputs.as_slice());
```

Comment:
don't you need to take ownership of the inputs here?

Reply:
They are cloned there internally. It should be ok since we clear the inputs vector later.